.circleci/config.yml

```diff
@@ -251,21 +248,14 @@ jobs:
     steps:
       - checkout
       - run:
-          name: build_docker image
+          name: Build and push Docker image
           no_output_timeout: "1h"
           command: |
-            cd .circleci/smoke_test/docker && docker build . -t ${image_name}:${CIRCLE_WORKFLOW_ID}
-      - run:
-          name: upload docker image
-          no_output_timeout: "1h"
-          command: |
-            set +x
-            export AWS_ACCESS_KEY_ID=${ECR_AWS_ACCESS_KEY}
-            export AWS_SECRET_ACCESS_KEY=${ECR_AWS_SECRET_ACCESS_KEY}
-            eval $(aws ecr get-login --region us-east-1 --no-include-email)
             set -x
-            docker tag ${image_name}:${CIRCLE_WORKFLOW_ID} 308535385114.dkr.ecr.us-east-1.amazonaws.com/${image_name}:${CIRCLE_WORKFLOW_ID}
-            docker push 308535385114.dkr.ecr.us-east-1.amazonaws.com/${image_name}:${CIRCLE_WORKFLOW_ID}
+            cd .circleci/smoke_test/docker && docker build . -t ${image_name}:${CIRCLE_WORKFLOW_ID}
```

Comment:
```suggestion
            echo "${DOCKER_HUB_TOKEN}" | docker login --username "${DOCKER_HUB_USERNAME}" --password-stdin
            cd .circleci/smoke_test/docker && docker build . -t ${image_name}:${CIRCLE_WORKFLOW_ID}
```

Reply:
To actually use these variables you'll need to also add the `org-member` security context to the build job for these.

Reply:
Applied the suggestions.